example/yarn.lock

```diff
@@ -2592,11 +2592,23 @@ array-includes@^3.1.5, array-includes@^3.1.6:
     get-intrinsic "^1.1.3"
     is-string "^1.0.7"
 
+array-union@^1.0.1:
```

Comment:
hmm, what do you think about `yarn-deduplicate`?
should we do it for `example`? or it sounds like overkill? ;)

Reply:
good practice - why not)

Reply:
no changes were present after running